app/components/DropdownMenu/DropdownMenu.js

```diff
@@ -40,18 +46,24 @@ class DropdownMenu extends React.Component<Props> {
       invariant(document.body, 'why you not here');
 
       if (currentTarget instanceof HTMLDivElement) {
-        const bodyRect = document.body.getBoundingClientRect();
-        const targetRect = currentTarget.getBoundingClientRect();
-        this.top = targetRect.bottom - bodyRect.top;
-
-        if (this.props.position === 'left') {
-          this.left = targetRect.left;
-        } else if (this.props.position === 'center') {
-          this.left = targetRect.left + targetRect.width / 2;
-        } else {
-          this.right = bodyRect.width - targetRect.left - targetRect.width;
+        this.bodyRect = document.body.getBoundingClientRect();
+        this.labelRect = currentTarget.getBoundingClientRect();
+        this.top = this.labelRect.bottom - this.bodyRect.top;
+        this.bottom = undefined;
+        this.position = this.props.position || 'left';
+
+        if (currentTarget.parentElement) {
+          const triggerParentStyle = getComputedStyle(
+            currentTarget.parentElement
+          );
+
+          if (triggerParentStyle.position === 'static') {
+            this.fixed = true;
```

Comment:
```suggestion
            this.fixed = true;
            this.top = this.labelRect.bottom;
```

Hey @MadMango – testing this out there still seems to be an issue with the positioning of fixed menus when the body is scrolled – but adding this additional line fixes it for me